elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -24,8 +32,10 @@
  */
 public class Util {
 
-    public static EntityManagerFactory getEntityManagerFactory(String modelPackageName, boolean includeAsyncModel, 
-            Properties options) {
+    public static ElideDynamicEntityCompiler dynamicEntityCompiler;
+
+    public static EntityManagerFactory getEntityManagerFactory(String modelPackageName, boolean includeAsyncModel,
+            boolean includeDynamicModel, String dynamicConfigPath, Properties options) throws Exception {
```

Comment:
Instead of includingDynamicModel - maybe have an` Optional<DynamicEntityCompiler>`